charger/easee.go

```diff
@@ -280,6 +281,50 @@ func (c *Easee) ProductUpdate(i json.RawMessage) {
 		// received observation is outdated, ignoring
 		return
 	}
+
+	switch res.ID {
```

Comment:
Please just integrate into the larger switch block below.

Reply:
ok